test/lib/jdk/test/lib/security/SecurityUtils.java

```diff
@@ -60,6 +61,19 @@ public static void removeFromDisabledTlsAlgs(String... protocols) {
         removeFromDisabledAlgs("jdk.tls.disabledAlgorithms",
                                List.<String>of(protocols));
     }
+    
+    public static void addToDisabledTlsAlgs(String... protocols) {
```

Comment:
Move these to line 55 (before removeFromDisabledTlsAlgs method) so remove and add methods stay together.